cli/localci/cli/main.py

```
@@ -20,14 +21,64 @@
+    """Print a user-friendly message for an unhandled internal error."""
+    display_path = str(log_path.expanduser())
+    messages = [
+        "An unexpected internal error occurred.",
+        f"{type(exc).__name__}: {exc}",
+        f"Please file a bug report and attach {display_path} "
+        "(contains the full traceback).",
+    ]
+    try:
+        for msg in messages:
+            print_error(msg)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)